src/wallet/asyncrpcoperation_sendmany.cpp

```diff
@@ -503,7 +550,25 @@ uint256 AsyncRPCOperation_sendmany::main_impl() {
 std::pair<uint256, uint256> AsyncRPCOperation_sendmany::SelectOVKs(const SpendableInputs& spendable) const {
     uint256 internalOVK;
     uint256 externalOVK;
-    if (!spendable.saplingNoteEntries.empty()) {
+    if (!spendable.orchardNoteMetadata.empty()) {
+        std::optional<OrchardFullViewingKey> fvk;
+        std::visit(match {
+            [&](const libzcash::UnifiedFullViewingKey& ufvk) {
+                fvk = ufvk.GetOrchardKey().value();
+            },
+            [&](const AccountZTXOPattern& acct) {
+                auto ufvk = pwalletMain->GetUnifiedFullViewingKeyByAccount(acct.GetAccountId());
+                fvk = ufvk.value().GetOrchardKey().value();
```

Comment:
Similarly, why is this guaranteed to have a value?

Reply:
This section is conditional on `!spendable.orchardNoteMetadata.empty()`, meaning we selected Orchard notes, and `ztxoSelector` must therefore have an Orchard component.